docs/reference-use-cases.md

```diff
@@ -0,0 +1,215 @@
+  * [Operator of Management Cluster](#operator-of-management-cluster)
+  * [Multi-Cluster/Multi-Provider](#multi-cluster/multi-provider)
+  * [Provider Implementors](#provider-implementors)
+
+## Role Glossary
+- __User__: consumer of a Kubernetes-conformant cluster created by the Cluster API.
+    - Does not use Cluster API.
+- __Operator__: Administrator responsible for creating and managing a Kubernetes cluster deployed by Cluster API.
+    - Uses Cluster API.
+- __Multi-cluster operator__: An operator responsible for multiple Kubernetes clusters deployed by Cluster API.
+    - Uses Cluster API.
+    - Cares about keeping config similar between many clusters.
+
+![Cluster API Use Cases](https://user-images.githubusercontent.com/3118335/56234753-d639e980-603a-11e9-8d45-944c547a3280.png)
+
+## Icon Glossary
+- 🔭 Out of Scope for Cluster API itself, but should be possible via higher level tool.
+    - These are use-cases that we should take care not to prevent.
+
+## Operator of Workload Cluster
+
+### Creating Clusters
+
+- As an operator, given that I have a cluster running Cluster API, I want to be able to use declarative APIs to manage another Kubernetes cluster (create, upgrade, scale, delete).
+
+- As an operator, given that I have a cluster running Cluster API, I want to be able to use declarative APIs to manage a vendor’s Kubernetes conformant cluster (create, upgrade, scale, delete).
+
+- As an operator, when I create a new cluster using Cluster API, I want Cluster API to automatically create and manage any supporting provider infrastructure needed for my new cluster to function.
+
+- As an operator, when I create a new cluster using Cluster API, I want to be able to use existing infrastructure (e.g. VPC’s, SecurityGroups, veth, GPUs).
+
+- As an operator, when I create a new cluster using Cluster API, I want to be able to take advantage of resource aware topology (e.g. compute, device availability, etc.) to place machines.
+
+- As an operator, I need to have a way to make minor customizations before kubelet starts while using a standard node image and standard boot script. Example: write a file, disable hyperthreading, load a kernel module.
+
+- As an operator, I need to have a way to apply labels to Nodes created through ClusterAPI.  This will allow me to partition workloads across Nodes/Machines and MachineDeployments. Examples of labels include datacenter, subnet, and hypervisor, which applications can use in affinity policies.
+
+- As an operator, I want to be able to provision the nodes of a workload cluster on an existing vnet that I don’t have admin control of.
+
+### Specify Affinity and Antiaffinity Groups of Worker Nodes
+
+- As an operator, I want to be able to set affinity rules on the deployment of Nodes, making sure Machines are either colocated or not colocated on the same host, rack or other dimension. These rules may span multiple sets of Machines and MachineDeployments.
```

Comment:
This use case looks to me similar to the ones in above section, "Creating Clusters".